src/items.js

```diff
@@ -0,0 +1,217 @@
+import { v4 as uuidv4 } from 'uuid';
+import fetchPokemon from './api.js';
+import {
+  getElementById, createDivWithClass, createDivWithId,
+  createElement, createElementWithClass, createCommentButton, formatDate,
+} from './util.js';
+
+let items = null;
+
+const getItems = () => JSON.parse(localStorage.getItem('items'));
+
+const saveItems = (items) => localStorage.setItem('items', JSON.stringify(items));
```

Comment:
- [ ] As per project requirement, we should use the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) to record the different user interactions (likes, comments and reservations) instead of using the local storage. Kindly fix this issue.

- [ ] When a user clicks on the Like button of an item, the interaction should get recorded in the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270), and the screen gets updated.

- [ ] When the `Comments popup` loads, the web app is expected to retrieve data from the selected API and show details about the selected item. And likewise, retrieve data from the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) to show the item comments.

- [ ] When a user clicks on the "Comment" button, the data is expected to get recorded in the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) and the screen updated.

Reply:
Done.